### DISCUSSION TITLE
Sidebar

### DISCUSSION THREAD

**Discussion by titoff002** [themeselection/flyonui#24] [Ideas / Request new feature]
Hello,

First of all, very good work!

I have a question: there is a "navbar" component but I couldn't find a "SideBar" component. Are you going to add one? Otherwise, would you have any code/tips to give to create a “sidebar” like the one on your institutional site?

Thanks for your help.

Good day.

**Comment by PruthviPraj00** [comment]
Hello @titoff002,

Thank you for your kind words and support!

We have provided a sidebar example in the drawer component, which was built using collapse and drawer. We will let you know what the team decides on the individual sidebar component.

**Comment by PruthviPraj00** [comment]
Hello @titoff002,

You can checkout the Sidebar component in V1.2.0.
